spec/timezone.html

```diff
@@ -38,26 +31,23 @@ <h1>Time Zone Names</h1>
       <h1>
         IsValidTimeZoneName (
           _timeZone_: a String,
-        )
+        ): a Boolean
       </h1>
       <dl class="header">
         <dt>description</dt>
-        <dd>It returns *true* if _timeZone_ is an ASCII-case-insensitive match for a built-in time zone name and *false* otherwise.</dd>
+        <dd>The returned value is *true* if _timeZone_ is an ASCII-case-insensitive match for a built-in time zone name, and *false* otherwise.</dd>
```

Comment:
```suggestion
        <dd>The returned value is *true* if _timeZone_ is an ASCII-case-insensitive match for a supported time zone name, and *false* otherwise.</dd>
```
This also suggests that the name is a bit misleading.

Reply:
The name is because that's what it's named in ECMA-402 and in the Intl.Enumeration proposal. We are free to name it something else in 262, of course.